### PR TITLE
Add check args and print usage to st2ctl

### PR DESCRIPTION
**Pull request by m4dcoder**
Add basic check args to st2ctl. If check args fail, st2ctl will print usage information.

### CODE REVIEW

File: tools/st2ctl [hunk 1]
```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
+
+set -e
+
 LOGFILE="/tmp/st2_startup.log"
 COMPONENTS="actionrunner st2api sensor_container rules_engine mistral st2resultstracker"
 STANCONF="/etc/st2/st2.conf"
 PYTHON=`which python`
-if [ -z "$AR" ];
```
Comment (DoriftoShoes) [inline tools/st2ctl]: Did we add in the ability to set the number of action runners somewhere else?  The current configuration allows us to use envvars to do it.

Comment (m4dcoder) [inline tools/st2ctl]: No.  I was not clear from this script what that block was doing because $AR didn't seem to come from anywhere.

**Comment by dzimine** [comment]
LGTM, solves the problem (assuming it works :) :+1:

**Comment by m4dcoder** [comment]
Addressed @DoriftoShoes comment.  Yeah, I hand tested this on st2express.

**Pull request merged**